doc/classes/Projection.xml

```diff
@@ -131,152 +151,195 @@
 			<param index="6" name="intraocular_dist" type="float" />
 			<param index="7" name=" convergence_dist" type="float" />
 			<description>
+				Creates a new [Projection] that projects positions using a perspective projection with the given Y-axis field of view (in degrees), X:Y aspect ratio, and clipping distances. The projection is adjusted for a head-mounted display with the given distance between eyes and distance to a point that can be focused on.
+				[param eye] creates the projection for the left eye when set to 1, or the right eye when set to 2.
+				[param flip_fov] determines whether the projection's field of view is flipped over its diagonal.
 			</description>
 		</method>
 		<method name="determinant" qualifiers="const">
 			<return type="float" />
 			<description>
+				Returns a scalar value that is the signed factor by which areas are scaled by this matrix. If the sign is negative, the matrix flips the orientation of the area.
+				The determinant can be used to calculate the invertibility of a matrix or solve linear systems of equations involving the matrix, among other applications.
 			</description>
 		</method>
 		<method name="flipped_y" qualifiers="const">
 			<return type="Projection" />
 			<description>
+				Returns a copy of this [Projection] with the signs of the values of the Y column flipped.
 			</description>
 		</method>
 		<method name="get_aspect" qualifiers="const">
 			<return type="float" />
 			<description>
+				Returns the X:Y aspect ratio of this [Projection]'s viewport.
 			</description>
 		</method>
 		<method name="get_far_plane_half_extents" qualifiers="const">
 			<return type="Vector2" />
 			<description>
+				Returns the dimensions of the far clipping plane of the projection, divided by two.
 			</description>
 		</method>
 		<method name="get_fov" qualifiers="const">
 			<return type="float" />
 			<description>
+				Returns the horizontal field of view of the projection (in degrees).
 			</description>
 		</method>
 		<method name="get_fovy" qualifiers="static">
 			<return type="float" />
 			<param index="0" name="fovx" type="float" />
 			<param index="1" name="aspect" type="float" />
 			<description>
+				Returns the vertical field of view of the projection (in degrees) associated with the given horizontal field of view (in degrees) and aspect ratio.
 			</description>
 		</method>
 		<method name="get_lod_multiplier" qualifiers="const">
 			<return type="float" />
 			<description>
+				Returns the factor by the visible level of detail is scaled by this [Projection].
+				Usage is: [code]lod_size / (lod_distance * multiplier) &lt; threshold[/code].
```

Comment:
This entry is unclear to me. Should it say "Returns the factor by _which_"?

As for the code block, does this function return that equation as a result. Or does this function return just the multiplier part?

Reply:
It should indeed say "by which". Thanks for catching that typo!
I copied the code block from a comment in `projection.cpp`, and if I understand correctly, the function returns just the multiplier part. Do you have any suggestions for how that could be better written?